rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -134,20 +105,6 @@ public class Coordinator extends Application {
     private final LRAService lraService;
     private final RecoveryCoordinator recoveryCoordinator;
 
-    @Override
-    public Set<Class<?>> getClasses() {
```

Comment:
I would have preferred to add this comment to the beans.xml but I can't seem to do that (perhaps because it's empty).

Given this `getClasses` method is being removed from here, is it the addition of the empty beans.xml that allows for this?